.ycm_extra_conf.py

```diff
@@ -36,7 +36,7 @@
 # CHANGE THIS LIST OF FLAGS. YES, THIS IS THE DROID YOU HAVE BEEN LOOKING FOR.
 flags = [
     '-DNDEBUG',
-    '-DNMC_HAVE_TBB',
+    '-DNMC_HAVE_CTHREAD',
```

Comment:
It was to fix a long standing issue with clang code completion in vim... something that only I use. I could revert it.